src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -684,6 +692,9 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
                     throw Host.ExceptSchemaMismatch(nameof(inputSchema), "label", (string)labelColumn.Name, "Key", (string)labelType.ToString());
 
                 _classCount = labelCount == 1 ? 2 : (int)labelCount;
+
+                //Temporary string array for key values, will replace soon
+                _keyValueAnnotations = new string[] { "sunflowers", "roses", "dandelion", "daisy", "tulips"};
```

Comment:
>_keyValueAnnotations [](start = 16, length = 20)

It seems that the `loadModel` argument specifies whether we are training the model or loading it from a file, inside the if in line 718 is where you should extract the key values from the input schema.